src/main/resources/google_checks.xml

```diff
@@ -100,22 +100,25 @@
     <module name="RightCurly">
       <property name="id" value="RightCurlySame"/>
       <property name="tokens"
-               value="LITERAL_TRY, LITERAL_CATCH, LITERAL_FINALLY, LITERAL_IF, LITERAL_ELSE,
-                    LITERAL_DO"/>
+                value="LITERAL_TRY, LITERAL_CATCH, LITERAL_IF, LITERAL_DO"/>
```

Comment:
only LITERAL_FINALLY can be from same to alone.
ELSE/CATCH should stay in "same" and we can try to do some xpath modofictions to reduce amount false-negatives.

Reply:
ELSE/CATCH were by default in "same" so I reverted the changes and move them back again to "same" and removed them from the suppression. I moved FINALLY to "alone" at https://github.com/checkstyle/checkstyle/pull/15789, in that PR I didnt needed to modify the suppression, all tests were passing by default.

Because of moving ELSE/CATCH to "same" and removing suppression, tests started failing, I have kept failing tests intentionally, I will fix it once my other PR is merged.